nginx.conf

```diff
@@ -26,21 +37,8 @@ server {
             return 204;
         }
 
-        add_header 'Access-Control-Expose-Headers' 'Content-Length,Content-Range';
-
-        root /usr/share/nginx/html;
-        try_files $uri $uri/ /index.html;
-
         location ~ \.json$ {
-            add_header 'Access-Control-Allow-Origin' "$http_origin" always;
-            add_header 'Access-Control-Allow-Methods' 'GET, OPTIONS';
-            add_header 'Access-Control-Allow-Headers' 'DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range';
             add_header Cache-Control no-cache;
         }
     }
-
-    error_page 500 502 503 504 /50x.html;
-    location = /50x.html {
-        root html;
-    }
```

Comment:
Don't we want custom 500 errors anymore?

Reply:
I didn't notice the `50x.html` anywhere, so figured it might be a mistake